### ISSUE TITLE
Unable to open/view pdf.

### ISSUE THREAD

**Issue by mjkoder** [manjit-hub/Notorite#69]
![image](https://github.com/user-attachments/assets/f04a8c5c-eec2-47a4-8580-e25f807b41ea)
![image](https://github.com/user-attachments/assets/bebe70a3-3766-488f-953b-c47579c86de2)

1. Unable to display uploaded pdf files from profile page.
2. Unable to display pdf from search in tab.

Fix these.

**Comment by Dhruv80576** [comment]
I would like to work on it. Will try to update as soon as possible

**Comment by Satvik1769** [comment]
you have to do this on your cloudinary website:

https://support.cloudinary.com/hc/en-us/articles/360016480179-PDF-or-ZIP-files-appearing-in-Media-Library-but-download-URLs-return-an-error

cheers

**Comment by Shivansh-0403** [comment]
Hello @manjit-hub 
When I worked on the Notes Controller issue, I also solved the problem of uploading pdfs to cloudinary and in my PR, I mentioned to change the Settings on Cloudinary in order to view pdfs as by default Cloudinary restricts the viewing of PDFs. 
Go to Cloudinary>Settings>Security>PDF and ZIP files delivery(At the Bottom)>Enable it.
Try this and if its still not working @Dhruv80576 can surely work on it and I would also be ready to share any insight or work on this issue.

![image](https://github.com/user-attachments/assets/28fb6c70-24bd-42ef-85b5-447f02848b69)
